### PR TITLE
Tf 01 002 navbar

### PR DESCRIPTION
**Pull request by Rasheduluctit**
The navbar was Changed and Updated with tailwind css. 
[Navbar.webm](https://github.com/UCT-IT/edume/assets/149924448/b6af9e05-8d4d-42d4-8798-dc7bf5b8608f)
Navbar-small
Navbar-large

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -16,7 +17,10 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        <Navbar></Navbar>
```
Comment (saminsss) [inline src/app/layout.tsx]: Self closing tag

File: src/components/home/Navbar/DropdownMenu.tsx [hunk 2]
```diff
@@ -10,42 +10,27 @@ type DropdownMenuPropsType = {
...
-          spacing={[4, 2, 2, 4]}
-          px={["20px", "10px", "20px", "30px"]}
-          py={["null", "10px", "15px", "20px"]}
-          backgroundColor="secondary"
-        >
+        <ul className="md:absolute md:top-full w-full md:w-64 flex-col items-start space-y-5 px-10 py-5 bg-secondary text-textColor">
           {dropdownMenus?.map((menu, i) => (
-            <Text key={i} _hover={{ fontWeight: "semibold", color: "122634" }}>
+            <li
+              key={i}
```
Comment (saminsss) [inline src/components/home/Navbar/DropdownMenu.tsx]: Shouldn't use index for keys

**Review by saminsss — CHANGES_REQUESTED** [review]

File: src/components/home/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -50,130 +52,88 @@ const Navbar = () => {
...
             <DropdownMenu {...dropdownSolution} />
             <Link
-              as={NextLink}
-              color="secondary"
               href="#"
-              fontSize={["null", "12px", "14px", "16px"]}
-              _hover={{ textDecoration: "none" }}
+              className="text-secondary hover:no-underline hover:text-[#122634]"
             >
               Testimonials
```
Comment (saminsss) [inline src/components/home/Navbar/Navbar.tsx]: Should be coming in from Locales

**Comment by saminsss** [comment]
Can you add more padding-top/bottom (4px) for menu dropdowns for the lg screens

**Comment by saminsss** [comment]
This looks weird. Should show a background once scrolled

Screenshot 2023-11-19 at 9 54 42 PM

File: src/app/layout.tsx (hunk 1 above)
Comment (Rasheduluctit) [reply src/app/layout.tsx]: done

File: src/components/home/Navbar/DropdownMenu.tsx (hunk 2 above)
Comment (Rasheduluctit) [reply src/components/home/Navbar/DropdownMenu.tsx]: Then I am using the menu as the key.

File: src/components/home/Navbar/Navbar.tsx (hunk 3 above)
Comment (Rasheduluctit) [reply src/components/home/Navbar/Navbar.tsx]: done

**Comment by Rasheduluctit** [comment]
scroll effect was showing when it scrolled down 200px. Now I have changed it to 20px. I hope Now It will be fine.

**Review by saminsss — APPROVED** [review]

**Pull request merged**